lc0_main.go

```diff
@@ -100,6 +102,7 @@ func getExtraParams() map[string]string {
 		"user":     *user,
 		"password": *password,
 		"version":  "17",
+		"id":       strconv.Itoa(randId),
```

Comment:
I find "id" a bit misleading. Maybe "token" is better?

Reply:
Done.